.travis.yml

```diff
@@ -1,23 +1,40 @@
 language: scala
+
 scala:
+  - 2.12.2-bin-typelevel-4
+  - 2.11.11-bin-typelevel-4
```

Comment:
What is the purpose of compiling 2.12 and 2.11 with typelevelc while we still need to build 2.10 with lightbend?